tools/python_api/src_cpp/pyarrow/pyarrow_scan.cpp

```diff
@@ -17,6 +18,7 @@ namespace kuzu {
 PyArrowScanConfig::PyArrowScanConfig(const common::case_insensitive_map_t<Value>& options) {
```

Comment:
I wonder why PyArrowScan and PandasScan have different ways to convert scanConfig. I need to take another look to see if we can unify them. but can you also take a look?

Reply:
I think the reason is that we don't support the pyarrow scan options (`skipNum` and `limitNum`) in pandas scan so before this PR we don't actually support any configuration for pandas scan. I could look into if it's doable to support those options in pandas scan and if it is I'll add the support and unify the config in a separate PR.